templates/tftmpl/integration_test.go

```diff
@@ -229,14 +229,26 @@ func TestRenderTFVarsTmpl(t *testing.T) {
 			}
 			tmpl := hcat.NewTemplate(input)
 
-			ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+			ctx, cancel := context.WithTimeout(context.Background(), time.Second*8)
 			defer cancel()
+
+			gld, err := ioutil.ReadFile(tc.goldenFile)
```

Comment:
This will help more than anything as it should give Consul more time to fully register everything which is probably the issue.

Reply:
Good point! I also noticed that the e2e test has a similar Consul registration setup. It doesn't have similar flakiness though probably since it has a buffer when CTS starts up, similar to this ReadFile.